### PR TITLE
pvremove: add page

### PR DESCRIPTION
**Pull request by jackiesogi**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known): LVM version: 2.03.23(2) (2023-11-21) Library version: 1.02.197 (2023-11-21)**

### CODE REVIEW

File: pages/linux/pvremove.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# pvremove
+
+> Remove LVM labels from physical volume(s).
+> More information: <https://man7.org/linux/man-pages/man8/pvremove.8.html>.
+
+- Remove LVM label from a physical volume:
```
Comment (gutjuri) [inline pages/linux/pvremove.md:6]:
```suggestion
- Remove a LVM label from a physical volume:
```

File: pages/linux/pvremove.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`sudo pvremove {{physical_volume}}`
+
+- Display detailed output during the operation:
+
+`sudo pvremove --verbose {{physical_volume}}`
+
+- Remove LVM label without asking for confirmation:
```
Comment (gutjuri) [inline pages/linux/pvremove.md:14]:
```suggestion
- Remove a LVM label without asking for confirmation:
```

File: pages/linux/pvremove.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`sudo pvremove --yes {{physical_volume}}`
+
+- Forcefully remove LVM label:
```
Comment (gutjuri) [inline pages/linux/pvremove.md:18]:
```suggestion
- Forcefully remove a LVM label:
```

**Review by gutjuri — APPROVED** [review]
Thanks! LGTM after my suggestions.

**Review by spageektti — APPROVED** [review]

File: pages/linux/pvremove.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`sudo pvremove --yes {{physical_volume}}`
+
+- Forcefully remove a LVM label:
+
+`sudo pvremove --force {{physical_volume}}`
+
+- Display output in JSON format:
+
+`sudo pvremove --reportformat json {{physical_volume}}`
```
Comment (spageektti) [inline pages/linux/pvremove.md:24]:
```suggestion
`sudo pvremove --verbose {{/dev/sdaXY}}`

- Remove a LVM label without asking for confirmation:

`sudo pvremove --yes {{/dev/sdaXY}}`

- Forcefully remove a LVM label:

`sudo pvremove --force {{/dev/sdaXY}}`

- Display output in JSON format:

`sudo pvremove --reportformat json {{/dev/sdaXY}}`
```

**Review by spageektti — CHANGES_REQUESTED** [review]

File: pages/linux/pvremove.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`sudo pvremove --yes {{/dev/sdaXY}}`
+
+- Forcefully remove a LVM label:
+
+`sudo pvremove --force {{/dev/sdaXY}}`
+
+- Display output in JSON format:
+
+`sudo pvremove --reportformat json {{/dev/sdaXY}}`
```
Comment (spageektti) [inline pages/linux/pvremove.md:24]:
```suggestion
`sudo pvremove --verbose {{/dev/sdXY}}`

- Remove a LVM label without asking for confirmation:

`sudo pvremove --yes {{/dev/sdXY}}`

- Forcefully remove a LVM label:

`sudo pvremove --force {{/dev/sdXY}}`

- Display output in JSON format:

`sudo pvremove --reportformat json {{/dev/sdXY}}`
```

**Review by spageektti — APPROVED** [review]

**Pull request merged**